app/views/admin/cx_collections/index.html.erb

```diff
@@ -60,10 +60,10 @@
     <tr>
       <td class="text-base text-sm">
         for
-        <%- if @quarter != 0 %>
```

Comment:
Should this second row not appear if both Year and Quarter are "All"? The bare "for" looks odd.

<img width="705" alt="Screenshot 2024-09-24 at 10 22 15 AM" src="https://github.com/user-attachments/assets/51f61945-ac0d-453a-9dff-8527cf5894c2">

Reply:
<img width="524" alt="Screenshot 2024-10-16 at 10 33 36 AM" src="https://github.com/user-attachments/assets/e7c147e6-9767-452b-a177-e8e62155db93">